plugins/rhacm-backend/src/providers/ManagedClusterProvider.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ResourceEntity } from '@backstage/catalog-model';
+import * as winston from 'winston';
+import { Config } from '@backstage/config';
+import {
+  EntityProvider,
+  EntityProviderConnection,
+} from '@backstage/plugin-catalog-node';
+import {
+  ANNOTATION_ORIGIN_LOCATION,
+  ANNOTATION_LOCATION,
+} from '@backstage/catalog-model'
+import { CustomObjectsApi } from "@kubernetes/client-node"
+import { getManagedCluster, getManagedClusters, hubApiClient } from '../helpers/kubernetes';
+import { CONSOLE_CLAIM, HUB_CLUSTER_NAME_IN_ACM } from '../constants';
+import { getClaim } from '../helpers/parser';
+import { getHubClusterName } from '../helpers/config';
+
+/**
+ * Provides entities from AWS EKS Cluster service.
+ */
```

Comment:
I think this should change to:

```suggestion
/**
 * Provides entities from OpenShift ACM service.
 */
```
or something similar since we are not actually using `AWS EKS` in this plugin.

Reply:
Definitely! :+1: